### DISCUSSION TITLE
Add support for Asciidoc

### DISCUSSION THREAD

**Discussion by sidey79** [microsoft/PSDocs#158] [Ideas]
**Is your feature request related to a problem?**

There are serval flavours of markdown. Using asciidoc seems to be a more clear approch for documentation, wich has some more features.

Can you support output in asciidoc format in addition of markdown?

Migrating the generated markdown always via a converter to asciidoc or include markdown direcly into markdown is maybe an alternative, but it's not sure, that the converter supports your flavor of markdown.

> **Please vote up if this feature is a feature you are also interested in.**

**Comment by ElinoraWhite** [comment]
@sidey79 Thanks for you feature request. I don't see why this can't be supported in the future.

**Comment by ElinoraWhite** [comment]
Let transfer it to discussions and see if it is also desired by the community.
